manifests/profiles/demo.yaml

```diff
@@ -8,6 +8,10 @@ spec:
       envoyOtelAls:
         service: otel-collector.istio-system.svc.cluster.local
         port: 4317
+    - name: skywalking
+      skywalking:
+        service: skywalking-oap.istio-system.svc.cluster.local
```

Comment:
is this should be `tracing.istio-system.svc.cluster.local`?
https://github.com/istio/istio/blob/f98481879ae4f869478ad900af6c96db9b8593bd/samples/addons/extras/skywalking.yaml#L37